cmd/bucket-replication-handlers.go

```diff
@@ -232,10 +232,10 @@ func (api objectAPIHandlers) GetBucketReplicationMetricsHandler(w http.ResponseW
 	enc := json.NewEncoder(w)
 	stats := globalReplicationStats.getLatestReplicationStats(bucket)
 	bwRpt := globalNotificationSys.GetBandwidthReports(ctx, bucket)
-	bwMap := bwRpt.BucketStats[bucket]
+	bwMap := bwRpt.BucketStats
```

Comment:
`GetBucketReplicationMetricsHandler` will get a bucket metric. It seems it will Deprecated?

Reply:
yeah since the `bucket` is already specified in GetBandwidthReports() in this PR this has been re-organized more cleanly.